lib/taskbook.js

```diff
@@ -325,8 +325,50 @@ class Taskbook {
     render.successCreate(task);
   }
 
+  _flattenDown(array, result) {
+    for (let i = 0; i < array.length; i++) {
+      const value = array[i];
+
+      if (Array.isArray(value)) {
+        this._flattenDown(value, result);
+      } else {
+        result.push(value);
+      }
+    }
+
+    return result;
+  }
+
+  _generateNumbers(value) {
```

Comment:
Maybe rename to _generateNumbersFromInterval
```suggestion
  _generateNumbersFromInterval(value) {
```